Sources/Apodini/Configurations/HTTPConfiguration.swift

```diff
@@ -41,11 +41,11 @@ public final class HTTPConfiguration: Configuration {
     }
     
     
-    /// initalize HTTPConfiguration
+    /// initialize HTTPConfiguration
     /// - Parameters:
     ///   - hostname: The `Hostname` that is used for populate information in exporters, the default value is `localhost:80` if there is no TLS configuration passed in the `HTTPConfiguration`, port 443 otherwise.
     ///   - bindAddress: The `BindAddress` that is used for bind the web service to a network interface, the default value is `0.0.0.0:80` if there is no TLS configuration passed in the `HTTPConfiguration`, port 443 otherwise.
-    ///   - tlsConfiguration: Information about the key and certificate needed to enable HTTPS.
+    ///   - tlsConfigurationBuilder: Information about the key and certificate needed to enable HTTPS.
```

Comment:
There seems to be a regression in the documentation here. We have renamed the property to `tlsConfiguration `.

Reply:
Afaik, swift documentation always uses the inner parameter name for the documentation. Either that or my IDE warning system is broken and confuses those two 🤔